azure-mgmt-compute/src/main/java/com/microsoft/azure/management/compute/DiskSkuTypes.java

```diff
@@ -24,6 +24,12 @@
     /** Static value Classic for DiskSkuTypes. */
     public static final DiskSkuTypes PREMIUM_LRS = new DiskSkuTypes(DiskStorageAccountTypes.PREMIUM_LRS);
 
+    /** Static value Classic for DiskSkuTypes. */
+    public static final DiskSkuTypes STANDARD_SSD_LRS = new DiskSkuTypes(DiskStorageAccountTypes.STANDARD_SSD_LRS);
+
+    /** Static value Classic for DiskSkuTypes. */
```

Comment:
The javadoc on static seems off.

Those on .NET

```
        /// <summary>
        /// Static value StandardLRS for DiskSkuTypes.
        /// </summary>
        public static readonly DiskSkuTypes StandardLRS = new DiskSkuTypes(DiskStorageAccountTypes.StandardLRS);
        /// <summary>
        /// Static value PremiumLRS for DiskSkuTypes.
        /// </summary>
        public static readonly DiskSkuTypes PremiumLRS = new DiskSkuTypes(DiskStorageAccountTypes.PremiumLRS);
        /// <summary>
        /// Static value StandardSSD_LRS for DiskSkuTypes.
        /// </summary>
        public static readonly DiskSkuTypes SStandardSSDLRS = new DiskSkuTypes(DiskStorageAccountTypes.StandardSSDLRS);
        /// <summary>
        /// Static value UltraSSDLRS for DiskSkuTypes.
        /// </summary>
        public static readonly DiskSkuTypes UltraSSDLRS = new DiskSkuTypes(DiskStorageAccountTypes.UltraSSDLRS);
```

Reply:
Updated.